### PR TITLE
Code Changes for PR #52

### PR DESCRIPTION
**Pull request by claireyoung**
I did several passes. Hope it's right this time - I keep on finding extra things here and there, but I think this is mostly it.

### CODE REVIEW

**Comment by claireyoung** [comment]
That's weird. My tests all pass in both Android Studio's Junit and ./gradlew build.

File: query/README.md [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
-Double minimum = queryClient.minimum("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double maximum = queryClient.maximum("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double average = queryClient.average("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double median = queryClient.median("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double percentile = queryClient.percentile("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
-Double sum = queryClient.sum("<event_collection>", "<target_property>", new RelativeTimeframe("this_year"));
+long count = queryClient.count("<event_collection>", new RelativeTimeframe("this_week"));
+long countUnique = queryClient.countUnique("<event_collection>", "<target_property>", new AbsoluteTimeframe("2015-05-15T19:00:00.000Z","2015-06-07T19:00:00.000Z"));
+double minimum = queryClient.minimum("<event_collection>", "<target_property>", new RelativeTimeframe("this_week"));
```
Comment (Geeber) [inline query/README.md]: Minor concern: technically, minimum/maximum/average/median can return null (if there are no events matching the timeframe & filters). This is a bit of an edge case, and maybe you're handling it some other way (e.g. by throwing an exception) somewhere else in the code, but it's something to think about.

File: query/src/test/java/io/keen/client/java/KeenQueryTest.java [hunk 2]
```diff
@@ -93,21 +88,42 @@ public void testCount()  throws Exception {
         assertEquals(requestString, "{\""+KeenQueryConstants.TIMEFRAME+"\":\"this_year\",\""+KeenQueryConstants.EVENT_COLLECTION+"\":\""+TEST_EVENT_COLLECTION+"\"}");
     }
 
+    public class Node {
```
Comment (Geeber) [inline query/src/test/java/io/keen/client/java/KeenQueryTest.java]: Question: what's this here for? I don't see it being used anywhere...

**Comment by Geeber** [comment]
Looks good to me, aside from the one issue about null's from minimum/maximum/etc. As I said it's kind of an edge case so I'd be OK shipping this beta as-is, and worry about it later. Let me know what you think. Whenever you're ready we can go ahead and merge :)

**Comment by claireyoung** [comment]
If null is a valid return value from the server, then perhaps the
execute() method should be allowed to sometimes return a null. Perhaps
queryResultToLong() and queryResultToDouble() - which are only used by
the easy count/max/min/etc. methods - should throw NullPointerException
with a custom message if the result is null?

**Comment by Geeber** [comment]
Hmm, the user experience around nulls still seems a little weird, but like I said it's a bit of an edge case. If you're happy with it then I'd say let's ship it and see what feedback we get. Sound good?
